src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -8291,7 +8282,6 @@ namespace VariableSpeedCoils {
         // Data is moved from the HP data structure to the HP outlet nodes.
 
         // Using/Aliasing
-        auto &TimeStepSys = state.dataHVACGlobal->TimeStepSys;
```

Comment:
Here's another that should be Real64 TimeStepSys = ...

Reply:
I replaced it as follows: `ReportingConstant = state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;`